src/app/project/project-routing.module.ts

```diff
@@ -89,33 +86,33 @@ const routes: Routes = [
         component: ProjectDetailComponent,
       },
       {
-        path: 'project-documents',
+        path: 'project-files',
         component: ProjectDocumentsComponent,
         resolve: {
           documents: DocumentsResolver
         }
       },
       {
-        path: 'project-documents/upload',
+        path: 'project-files/upload',
```

Comment:
Hopefully this won't break anything, is this dynamic only, or did this every get stored in a db.

Reply:
This is just like changing a route in react-router. Anything you see in a routing.module.ts is app routing. We don't store any app routes in the DB